internal/mountinfo/fixtures/generate-fixtures.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
TBH this seems quite hard to inspect and is a lot of vendored code for essentially just code which is writing out things with specific file names and symlinks. I am guessing your sys.ttar file is from a real sys dir. However, I suspect you would get a much more understandable test if you just hardcoded a few example layouts rather than one full sys dump.